cmd/scale.go

```diff
@@ -374,8 +375,8 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 				return errors.Wrap(err, "failed to get VMSS list in the resource group")
 			}
 			for _, vmss := range vmssListPage.Values() {
-				vmName := *vmss.Name
-				if !sc.vmInAgentPool(vmName, vmss.Tags) {
+				vmssName := *vmss.Name
```

Comment:
This is a code hygiene change: we are re-using the VMSS name as a variable, not the name of an individual vm instance. Naming the variable `vmssName` is clearer.